UnityProject/Assets/Scripts/Messages/Client/DevSpawner/DevDestroyMessage.cs

```diff
@@ -40,6 +40,13 @@ private void ValidateAdmin(NetMessage msg)
 				if (NetworkObject == null) return;
 
 				Vector2Int worldPos = NetworkObject.transform.position.RoundTo2Int();
+				if (NetworkObject.TryGetComponent<PlayerScript>(out var victim))
```

Comment:
I think we have an smite action, is that not working anymore? If it is, maybe this message should call that action and return, so we only have one way to destroy players.

Reply:
it's not a message it's a CMD request

![image](https://user-images.githubusercontent.com/34368774/182043026-a9fa9768-46d7-490c-a77c-65aa5334c687.png)

This message im editing is already being processed on the server so we don't need to do extra stuff